source/uri-options/tests/compression-options.yml

```diff
@@ -0,0 +1,44 @@
+tests:
+    -
+        description: "Valid compression options are parsed correctly"
+        uri: "mongodb://example.com/?compressors=zlib&zlibCompressionLevel=9"
+        valid: true
+        warning: false
+        hosts: ~
+        auth: ~
+        options:
+            compressors: "zlib"
```

Comment:
Since `compressors` is a 

> comma separated list of strings, e.g. “snappy,zlib”

I think this should be 1-member list with `"zlib"` as its only member.

Reply:
Nevermind, I suppose this isn't necessary (since it could just be a string with comma-separated values).